KathSchloe/CreativeU
# Backend stuffs (ks-cu-1)

Description
Added backend stuffs (seed database, create database, controllers, reops, models, and a components folder).

Testing Instructions
git add and git commit whatever you're working on on your branch
git checkout main
git fetch --a
git checkout <ks-cu-1>
If you don't see the most recent version of my files, git pull origin <ks-cu-1>
If you haven't created the Tabloid MVC database, please do so.
Make sure the databases and controllers, etc. are present.
Click the solid green triangle to run and confirm that the data is showing up in swagger.

Checklist:
 My code follows the style guidelines of this project
 I have performed a self-review of my code
 I have commented my code, particularly in hard-to-understand areas
 My changes generate no new warnings or errors
 I have added test instructions that prove my fix is effective or that my feature works
